src/Umbraco.Web.Common/Models/WebhookViewModel.cs

```diff
@@ -1,10 +1,13 @@
-﻿using System.Runtime.Serialization;
+using System.Runtime.Serialization;
 
 namespace Umbraco.Cms.Web.Common.Models;
 
 [DataContract]
 public class WebhookViewModel
 {
+    [DataMember(Name = "id")]
+    public int Id { get; set; }
+
```

Comment:
We don't want to leak ID's to the frontend (This is a concept from v14), as those are just implementation details.
We should use the key as an identifier (I guess that should be called Id 😛) As that is also consistent between environments 👍 

We do however still call it a "Key" on the back-end side, but we should call it "Id" for the frontend 👍

Reply:
Yes, I just noticed the GetAll() returned id and key property on the model, but it wasn't mapped here.

Do you want it for v13 for consistency and then remove it in v14?
We can obsolete the property?

The property can just be `Key` in v14 as well, the label above just show Id.

But in v14 we may also clean id properties in the views as well then?

Reply:
I think we should just remove Id here, and then Rename `Key` to Id (Still only on the ViewModel)
Then that should hopefully be an easy port to v14, as the ViewModel doesn't need to change 👍

Reply:
But why not just keep the name `Key` and listed in frontend as all other entities/models, like datatype, relation type etc.

The same markup is used in the info box.. if Id isn't present it just isn't shown.

But since webhook does have an Id in v13, I think it would make sense to show it in v13 and to be consistent, but just obsolete the property and remove it is in v14, but I guess all Id properties will be removed in v14.

Reply:
Webhooks will always have an ID, as it is a database implementation detail (maybe in the far future we will remove Id's entirely, who knows), we just don't wanna "leak" that information to the frontend 👍

But lets just keep it as is, and refactor for v14 👍